lib/repository.py

```diff
@@ -1,9 +1,15 @@
 """Checks the repository for updates."""
+from __future__ import print_function
+from __future__ import division
+from future import standard_library
+standard_library.install_aliases()
+from past.utils import old_div
+from builtins import range
 
 
 import os
 import sys
-import urllib
+import urllib.request, urllib.parse, urllib.error
```

Comment:
Python 2 has `urllib.urlopen()` and Python  3 has `urllib.request.urlopen()`.
The lines
```
from future import standard_library
standard_library.install_aliases()
```
gets and alternative version of `urllib` from the future package. This allows the syntax to be `urllib.request.urlopen()` everywhere.

Although, I agree that we don't need `urllib.parse` and `urllib.error`. I'll remove the unused imports everywhere.